spec/controllers/line_items_controller_spec.rb

```diff
@@ -60,8 +61,8 @@
         context "without an order cycle" do
           it "denies deletion" do
             delete :destroy, params
-            expect(response.status).to eq 403
-            expect { item.reload }.to_not raise_error
+            expect(response.status).to eq 401
```

Comment:
Exactly, the reason of the failure is related to some internal business logic / authorization (`403`) not authentication (`401`).

One thing that could lead to confusion is that in the HTTP status protocol definition the therm "Authorization" is referred to what we refer to "Authentication" in web world 😞 

`CanCan` abilities -> authorization layer `403`
`Devise` and friends -> authentication layer `401`

Or am I missing something regarding the spec? From what I understand a logged in user cannot delete a line item if it is not associated to an order cycle.

I'm sorry if I'm getting this wrong, but at least it's useful to me to understand OFN business logic 🙏